### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
after discussion with PM John Taing

### CODE REVIEW

**Comment by get-itips** [comment]
I was also working on this in another PR, I'll close mine.

closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8770

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ Users who participate in conversations that are part of the chat list in Teams m
 |Size of each file attachment | Less than 10 MB |
 |Number of inline images<sup>2</sup> |50   |
 
+> [!NOTE]
+  > There is a throttling limit of how many emails you can send to a channel which  is 6 emails per 10 seconds per channel per user and 8 emails per 10 seconds per tenant per user
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:96]:
```suggestion
> There is a throttling limit on how many emails you can send to a channel. The limit is six emails per ten seconds per channel per user and eight emails per ten seconds per tenant per user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes. Many thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 approved changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 @get-itips  I have approved the changes, can we merge to live? it seems they are some more customers that are currently facing this limits and will be good to have this public facing.

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**